README.md

```diff
@@ -41,32 +41,21 @@ Please see the [wiki](https://github.com/microsoft/fluentui/wiki).
 
 ## Using Fluent UI React
 
-### Starter apps
+### Create a new app
 
-To create a simple starter app using Fluent UI components, [install the latest LTS node.js](https://nodejs.org), then run:
+To create a simple React app  [install node.js](https://nodejs.org), then run:
 
-```
-npm init uifabric
-```
-
-It'll prompt you for a project name. If you choose to create a project called `my-app`, you can start working on the project like this:
-
-```
+```sh
+npx create-react-app my-app
 cd my-app
 npm start
```

Comment:
Before running `npm start`, we should move the instruction below to add fluentui up here:

```tsx
cd my-app
npm install @fluentui/react
npm start
```